### DISCUSSION TITLE
c++ 调用rust项目，rust的项目可以用cargo不用xmake吗

### DISCUSSION THREAD

**Discussion by holdxen** [xmake-io/xmake#5811] [Q&A]
https://xmake.io/#/zh-cn/guide/project_examples?id=%e4%bd%bf%e7%94%a8-cxxbridge-%e5%9c%a8-c-%e4%b8%ad%e8%b0%83%e7%94%a8-rust

文档提供了一种方式用来c++ rust交互，但是rust的项目是用xmake管理的。
我想用xmake管理c++项目，添加用cargo管理的rust库

类似这样的, 现在有办法实现吗?
 
![图片](https://github.com/user-attachments/assets/58a5778c-6dae-42f9-a413-bdddadf210a6)

**Comment by waruqi** [comment]
https://github.com/xmake-io/xmake/blob/dev/tests/projects/rust/cxx_call_rust_library/xmake.lua

**Reply by holdxen** [reply]
> https://github.com/xmake-io/xmake/blob/dev/tests/projects/rust/cxx_call_rust_library/xmake.lua

这个好像是用xmake管理rust的，我想用cargo去管理

**Reply by waruqi** [reply]
这个？https://docs.rs/xmake/latest/xmake/
